contracts/Token.sol

```diff
@@ -4,68 +4,62 @@ pragma solidity ^0.8.0;
 import "@openzeppelin/contracts-upgradeable/token/ERC20/extensions/ERC20PausableUpgradeable.sol";
 import "@openzeppelin/contracts-upgradeable/access/OwnableUpgradeable.sol";
 
-struct FrozenWallet {
-    address wallet;
-    uint totalAmount;
-    uint monthlyAmount;
-    uint initialAmount;
-    uint startDay;
-    uint afterDays;
-    bool scheduled;
-    uint monthDelay;
-}
+// FIXME: Solidity warning
+// FIXME: Indentation
+    struct FrozenWallet {
+        address wallet;
+        uint256 totalAmount;
+        uint256 monthlyAmount;
+        uint256 initialAmount;
+        uint256 lockDaysPeriod;
+        // TODO: why do we need this variable?
```

Comment:
which one?  below or above? :) @DenisGorbachev 

`        bool scheduled;` - is a special solidity trick to check if entity exists by extra bool value in the end.